### PR TITLE
doc: update CPU affinity related descriptions

### PR DESCRIPTION
**Pull request by ZideChen0**
- Changed term from "VCPU affinity" to "CPU affinity"
- changed vcpu_affinity to cpu_affinity_bitmap in vm_config
- fixed some errors

Signed-off-by: Zide Chen <zide.chen@intel.com>

### CODE REVIEW

File: doc/tutorials/acrn_configuration_tool.rst [hunk 1]
```diff
@@ -132,9 +132,8 @@ Additional scenario XML elements:
   The order of severity from high to low is:
   ``SEVERITY_SAFETY_VM``, ``SEVERITY_RTVM``, ``SEVERITY_SOS``, ``SEVERITY_STANDARD_VM``.
 
-``vcpu_affinity``:
-  vCPU affinity map. Each vCPU will be mapped to the selected pCPU ID. A different vCPU in the same VM cannot be mapped to the same pCPU.
-  If the pCPU is mapped by different VMs, ``cpu_sharing`` of the VM must be set to ``Enabled`` in the launch XML.
+``cpu_affinity``:
```
Comment (dbkinder) [inline doc/tutorials/acrn_configuration_tool.rst:135]: should this be ``cpu_affinity_bitmap``?

File: doc/tutorials/cpu_sharing.rst [hunk 2]
```diff
@@ -21,13 +21,12 @@ The below diagram shows that the vCPU layer invokes APIs provided by scheduling
 .. figure:: images/cpu_sharing_api.png
    :align: center
 
-vCPU affinity
+CPU affinity
 *************
 
-Currently, we do not support vCPU migration; the assignment of vCPU mapping to pCPU is statically configured in the VM configuration via a vcpu_affinity array. The item number of the array matches the vCPU number of this VM. Each item has one bit to indicate the assigned pCPU of the corresponding vCPU. Use these rules to configure the vCPU affinity:
+Currently, we do not support vCPU migration; the assignment of vCPU mapping to pCPU is fixed at the time the VM is launched. The statically configured cpu_affinity_bitmap in the VM configuration defines a superset of pCPUs that the VM is allowed to run on. One bit in this bitmap indicates that one pCPU could be assigned to this VM, and the bit number is the pCPU ID. A pre-launched VM is supposed to be launched on exact number of pCPUs that are assigned in this bitmap. and the vCPU to pCPU mapping is implicitly indicated: vCPU0 maps to the pCPU with lowest pCPU ID, vCPU1 maps to the second lowest pCPU ID, and so on.
```
Comment (dbkinder) [inline doc/tutorials/cpu_sharing.rst:27]: please keep line length < 80 characters, also delete the leading space on this sentence.

File: doc/tutorials/cpu_sharing.rst [hunk 3]
```diff
@@ -21,13 +21,12 @@ The below diagram shows that the vCPU layer invokes APIs provided by scheduling
...
 
-- Only one bit can be set for each affinity item of vCPU.
-- vCPUs in the same VM cannot be assigned to the same pCPU.
+For post-launched VMs, acrn-dm could choose to launch a subset of pCPUs that are defined in cpu_affinity_bitmap by specifying the assigned pCPUs (``--cpu_affinity`` option). But it can't assign any pCPUs that are not included in the VM's cpu_affinity_bitmap.
```
Comment (dbkinder) [inline doc/tutorials/cpu_sharing.rst:29]: delete leading space on this sentence (and keep line length < 80 characters)
Should the option be called ``--cpu_affinity_bitmap``?  (That's its name in the .c file)

Comment (ZideChen0) [reply doc/tutorials/cpu_sharing.rst:29]: From command line option's perspective, "cpu_affinity" seems accurate, which tell hypervisor to launch guest VM on these pCPUs. While we use "cpu_affinity_bitmap" in the . c code, which is to emphasize that it's a list of pCPUs that are possibly to assigned to the VM, but DM could only choose a subset from it.

File: doc/tutorials/cpu_sharing.rst (hunk 2 above)
Comment (ZideChen0) [reply doc/tutorials/cpu_sharing.rst:27]: There are a number of paragraphs in this doc are written in this style. I can change all of them and update the PR.

File: doc/tutorials/acrn_configuration_tool.rst (hunk 1 above)
Comment (ZideChen0) [reply doc/tutorials/acrn_configuration_tool.rst:135]: This is correct. In the .xml files, it's "cpu_affinity" for simplicity.

**Comment by ZideChen0** [comment]
BTW, I'd suggest not merge this PR for now. This PR is conflict with PR4754. I'll talk to @conghuic23 and see if it's going to release_1.6 as well.

**Pull request merged**